AppCenterCrashes/AppCenterCrashesTests/MSCrashesTests.mm

```diff
@@ -1203,6 +1205,54 @@ - (void)testCrashesSetCorrectUserIdToLogs {
   [settings stopMocking];
 }
 
+#if !TARGET_OS_OSX
+
+- (void)testObserverAddedNotOsxNotExtension {
```

Comment:
You can add one more test for extension by mocking `MS_IS_APP_EXTENSION`.
Then the name of this test would be like `testObserverAdded` since the macro explains the test isn't being executed for macOS platform.